terraform/amazon/network-existing-vpc/vpc_peering.tf

```diff
@@ -0,0 +1,59 @@
+data "terraform_remote_state" "vpc_peer_stack" {
```

Comment:
I think this file isn't necessary, all existing vpc cluster need to be in the same VPC, so no peering required